### DISCUSSION TITLE
feature request: list/search issues, where I am '@' mentioned.

### DISCUSSION THREAD

**Discussion by AdamJel** [ankitpokhrel/jira-cli#496] [Ideas]
Hi,

absolutely great feature would be finding all places/issues, where I am mentioned in comments. Would this be possible, please?

Thanks

**Comment by ankitpokhrel** [comment]
@AdamJel I believe you can do something like this?

```sh
// Get all issues where I am tagged in description
$ jira issue list -q "text ~ currentUser()"

// Get all issues where I am tagged in comments
$ jira issue list -q "comment ~ currentUser()"

// Get all issues where I am tagged in description or comments
jira issue list -q "text ~ currentUser() or comment ~ currentUser()"
```

**Reply by ankitpokhrel** [reply]
While this seem to be working, results may not be 100% accurate as per this open ticket in Jira: [JRACLOUD-27594](https://jira.atlassian.com/browse/JRACLOUD-27594)
